packages/test-e2e/test/__snapshots__/mocha-integration.tested.snap

```diff
@@ -2,4 +2,24 @@
 
 exports[`earljs/mocha failing test 1`] = `0.685109454492407`;
```

Comment:
what caused these snapshots to change :O?

Reply:
Newer Mocha version with `--parallel` created more snapshots. I just updated the file to be just what's happening after Mocha downgrade to v8.2